### PR TITLE
Fix #578 (tile-order/cell-order typo)

### LINKED ISSUE #578: cell & tile order swapped in SparseNdArray
There is a bug in SparseNdArray - it swaps the tile_order and cell_order assignment (see code below which is copied from the current `main`).  This has the effect of breaking the `platform_config` if the user tries to override one or the other.

```python
        sch = tiledb.ArraySchema(
            domain=dom,
            attrs=attrs,
            sparse=True,
            allows_duplicates=False,
            offsets_filters=create_options.offsets_filters(),
            capacity=create_options.get("capacity", 100000),
            tile_order=cell_order,
            cell_order=tile_order,
            ctx=self._ctx,
        )
```

### PR DESCRIPTION
**Pull request by johnkerl**
Fixes typo pointed out in #578

### CODE REVIEW

File: apis/python/src/tiledbsoma/dense_nd_array.py [hunk 1]
```diff
@@ -213,6 +214,8 @@ def _dense_index_to_shape(
 
     See read_tensor for usage.
     """
+    print("INPUT COORDS", coords)
+    print("INPUT SHAPE ", array_shape)
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/dense_nd_array.py:218]: Left-over debug output?

File: apis/python/src/tiledbsoma/dense_nd_array.py [hunk 2]
```diff
@@ -221,11 +224,19 @@ def _dense_index_to_shape(
             start, stop, step = idx.indices(array_shape[n])
             if step != 1:
                 raise ValueError("stepped slice ranges are not supported")
-            shape.append(stop - start)
+            #shape.append(stop - start)
+            # XXX
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/dense_nd_array.py:228]: Dead comment?

File: apis/python/src/tiledbsoma/dense_nd_array.py [hunk 3]
```diff
@@ -221,11 +224,19 @@ def _dense_index_to_shape(
...
+            if idx.stop is None:
+                shape.append(stop - start)
+            else:
+                shape.append(stop - start)
+            # XXX
         else:
             raise ValueError("coordinates must be tuple of int or slice")
 
     if result_order == "row-major":
+        print("OUTPUT SHAPE", tuple(shape))
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/dense_nd_array.py:238]: Left-over debug output?

File: apis/python/src/tiledbsoma/util.py [hunk 4]
```diff
@@ -127,3 +128,54 @@ def slice_to_range(ids: slice) -> Optional[Tuple[int, int]]:
...
+) -> Tuple[int, ...]:
+    """
+    Given a subarray index specified as a tuple of per-dimension slices or scalars
+    (eg, ``([:], 1, [1:2])``), and the shape of the array, return the shape of
+    the subarray.
+
+    See read_tensor for usage.
+    """
+    print("INPUT COORDS", coords)
+    print("INPUT SHAPE ", array_shape)
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/util.py:145]: Left-over debug output?

File: apis/python/src/tiledbsoma/util.py [hunk 5]
```diff
@@ -127,3 +128,54 @@ def slice_to_range(ids: slice) -> Optional[Tuple[int, int]]:
...
+    shape: List[int] = []
+    for n, coord in enumerate(coords):
+        shape.append(dense_index_to_shape(coord, array_shape[n]))
+
+    if result_order == "row-major":
+        print("OUTPUT SHAPE", tuple(shape))
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/util.py:151]: Left-over debug output?

File: apis/python/src/tiledbsoma/util.py [hunk 6]
```diff
@@ -127,3 +128,54 @@ def slice_to_range(ids: slice) -> Optional[Tuple[int, int]]:
...
+        return tuple(shape)
+
+    print("OUTPUT SHAPE", tuple(reversed(shape)))
```
Comment (eddelbuettel) [inline apis/python/src/tiledbsoma/util.py:154]: Left-over debug output?

**Review by eddelbuettel — COMMENTED** [review]
Seems like a few debug-print statements are left.

File: apis/python/src/tiledbsoma/dense_nd_array.py [hunk 7]
```diff
@@ -155,6 +155,7 @@ def read_tensor(
             target_shape = _dense_index_to_shape(coords, A.shape, result_order)
             query = A.query(return_arrow=True, order=tiledb_result_order)
             arrow_tbl = query.df[coords]
+            print("COORDS      ", coords)
```
[the remark this answers is not in the document]
Comment (johnkerl) [reply apis/python/src/tiledbsoma/dense_nd_array.py]: Fixed rebase

File: apis/python/src/tiledbsoma/dense_nd_array.py (hunk 2 above)
Comment (johnkerl) [reply apis/python/src/tiledbsoma/dense_nd_array.py]: Fixed rebase

File: apis/python/src/tiledbsoma/util.py (hunk 4 above)
Comment (johnkerl) [reply apis/python/src/tiledbsoma/util.py]: Fixed rebase

File: apis/python/src/tiledbsoma/util.py (hunk 6 above)
Comment (johnkerl) [reply apis/python/src/tiledbsoma/util.py]: Fixed rebase

**Review by bkmartinjr — APPROVED** [review]
LGTM.  thanks for the fast fix!

**Pull request merged**